app/screens/channel/channel_nav_bar/channel_title/index.js

```diff
@@ -3,7 +3,8 @@
 
 import {connect} from 'react-redux';
 
-import {getCurrentChannel, getMyCurrentChannelMembership} from 'mattermost-redux/selectors/entities/channels';
+import {getCurrentChannel} from 'mattermost-redux/selectors/entities/channels';
```

Comment:
These shouldn't need to be changed. `selectors/entities/channels` should be reexporting the common selectors so that the path to them doesn't change

Reply:
https://github.com/mattermost/mattermost-redux/pull/452

Reply:
yeah, I do agree, so lets have a PR in redux and I'll update this one

Reply:
@hmhealey @enahum I like the strategy you're proposing of re-exporting the common selectors. I have queued it for the upcoming dev meeting as an explicit policy proposal moving forward (or at least as a PSA if it's already a thing).